go.mod

```diff
@@ -59,14 +59,14 @@ require (
 	github.com/gorilla/websocket v1.5.3
 	github.com/jinzhu/copier v0.3.5
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/longhorn/backing-image-manager v1.6.4-rc1
+	github.com/longhorn/backing-image-manager v1.6.4
 	github.com/longhorn/backupstore v0.0.0-20241227145911-8de988de7eff
 	github.com/longhorn/go-common-libs v0.0.0-20241227132452-5a6ca245288a
```

Comment:
since you have https://github.com/TykTechnologies/longhorn-go-common-libs/pull/1 need to pull that lib instead of making the changes here directly.

Reply:
we are not using the common libs anymore. Just edited everything from vendors https://tyktech.slack.com/archives/C02SYKVK6HY/p1744318961306569?thread_ts=1744226580.652909&cid=C02SYKVK6HY